modules/floating-buttons/assets/js/floating-bars/frontend/handlers/floating-bars.js

```diff
@@ -91,32 +91,35 @@ export default class FloatingBarsHandler extends Base {
 		}
 	}
 
-	initEntranceAnimation( element, animation ) {
-		const { none } = this.getSettings( 'constants' );
-
-		const entranceAnimationControl = this.getResponsiveSetting( animation );
+	initEntranceAnimation() {
+		const { animated, ctaEntranceAnimation, ctaEntranceAnimationDelay, hasEntranceAnimation } = this.getSettings( 'constants' );
+		const entranceAnimationClass = this.getResponsiveSetting( ctaEntranceAnimation );
+		const entranceAnimationDelay = this.getResponsiveSetting( ctaEntranceAnimationDelay ) || 0;
+		const setTimeoutDelay = entranceAnimationDelay + 500;
```

Comment:
@mserino What does the `500` mean?

Reply:
@louiswol94 I needed a bit of buffer before the class was removed otherwise I would get a flickering animation